sdk/eventhub/azure-eventhub/tests/unittest/test_event_data.py

```diff
@@ -90,18 +90,18 @@ def test_sys_properties():
 
 
 def test_event_data_batch():
-    batch = EventDataBatch(max_size_in_bytes=100, partition_key="par")
+    batch = EventDataBatch(max_size_in_bytes=110, partition_key="par")
```

Comment:
@yunhaoling the size of the batch needs to be updated. Since every EventData object now has `MessageProperties()` added in the constructor automatically (even if all properties on MessageProperties are `None`), it increases size of each EventData. Previously, `message.properties = None` not `MessageProperties`.